test/run_tests.sh

```diff
@@ -163,6 +163,8 @@ echo "==========================================================================
 echo $'\t\t\tInstalling test requirements'
 echo "=================================================================================="
 pip3 install $UMODE_OPT -r $KEYLIME_DIR/test/test-requirements.txt
+git clone https://github.com/keylime/rust-keylime.git $KEYLIME_DIR/../rust-keylime
```

Comment:
Can we use a variable, that if is set will do the clone and the cargo build? Something line `RUST_TEST` set as 0 in the script but overwritten from the value in the environment, as `RUST_TEST="${RUST_TEST:-0}"`

This way by default we do not clone and call `cargo build` when running the Python tests.  There is no guarantee that the environment will even have Rust installed when building and testing a Python program.

Reply:
Very true - I've added this in my latest push, and gated the Rust tests and build behind it.